pom.xml

```diff
@@ -66,8 +66,8 @@
     <jbossws.spi.version>4.1.0.Final</jbossws.spi.version>
     <jbossws.common.version>4.0.0.Final</jbossws.common.version>
     <jbossws.common.tools.version>1.4.0.Final</jbossws.common.tools.version>
-    <wildfly2700.version>27.0.0.Alpha4</wildfly2700.version>
-    <wildfly27dev.version>27.0.0.Beta1-SNAPSHOT</wildfly27dev.version>
+    <wildfly2700.version>27.0.0.Beta1</wildfly2700.version>
+    <wildfly27dev.version>27.0.0.Final-SNAPSHOT</wildfly27dev.version>
```

Comment:
what about dropping the version from this property name completely to avoid need for updating CI with every new WF major?
something like..
` <wildflydev.version>27.0.0.Final-SNAPSHOT</wildflydev.version>`
` <wildfly.dev.version>27.0.0.Final-SNAPSHOT</wildfly.dev.version>`
` <wildfly.next.version>27.0.0.Final-SNAPSHOT</wildfly.next.version>`

Reply:
@jbliznak  That's a good idea. I created a jira for this and will look at wildfly version/profile soon.

Reply:
https://issues.redhat.com/browse/JBWS-4323